components/inputs/test/input-date.test.js

```diff
@@ -52,7 +52,7 @@ describe('d2l-input-date', () => {
 			const elem = await fixture(basicFixture);
 			const dropdown = getChildElem(elem, 'd2l-dropdown');
 			dropdown.toggleOpen();
-			await aTimeout(1);
+			await oneEvent(dropdown, 'd2l-dropdown-open');
```

Comment:
Yay less timeouts! I thought I remembered a timeout in some of the dropdown or menu auto-focusing code that was listening for `d2l-dropdown-open` event as well and then doing its own `setTimeout`.  I think that should be able to be removed?

Reply:
Do you have any more context on that? I'm seeing a lot of places where we setTimeout without a clear reason, so I'm not entirely sure what I should be looking for - though you're probably right. :)

Reply:
[This one](https://github.com/BrightspaceUI/core/blob/master/components/dropdown/dropdown-menu.js#L64) -- it was listening for that `d2l-dropdown-open` event and doing its own timeout (as is your scrollIntoView code in that same method). Now that the event is fired from inside a timeout, hopefully one or both of them aren't needed.

Reply:
The one you referenced can go, it works fine without it, but the timeout for scrolling things into view is still needed. Without it, the post-resize-scroll back to the top consistently happens after our desired scroll.